### PR TITLE
testnet

### PR DESCRIPTION
**Pull request by eitanz-coti**

### CODE REVIEW

File: src/crypto_utils.ts [hunk 1]
```diff
@@ -87,14 +87,30 @@ export function decryptRSA(privateKey: Uint8Array, ciphertext: string): string {
...
     }
 
     return userKey.join("")
 }
 
+export function recoverUserKey(privateKey: Uint8Array, encryptedKeyShare0: string, encryptedKeyShare1: string): string {
+    const decryptedKeyShare0: string = decryptRSA(privateKey, encryptedKeyShare0);
+    const decryptedKeyShare1: string = decryptRSA(privateKey, encryptedKeyShare1);
+
+    const bufferKeyShare0 = Buffer.from(decryptedKeyShare0, 'hex'); // 'hex' because decryptRSA is returning hex-encoded string
```
Comment (smiller-coti) [inline src/crypto_utils.ts]: We can't use Buffer in the browser/window environment

File: src/ethers_utils.ts [hunk 2]
```diff
@@ -29,7 +29,7 @@ export async function getAccountBalance(address: string, provider: Provider) {
     return provider.getBalance(address);
 }
 
-export function initEtherProvider(rpcUrl: string = "https://devnet.coti.io/rpc") {
+export function initEtherProvider(rpcUrl: string = "https://testnet.coti.io/rpc") {
```
Comment (smiller-coti) [inline src/ethers_utils.ts]: Do we want to remove support for devnet?

File: src/account/onboard-contract.ts [hunk 3]
```diff
@@ -1,40 +1,46 @@
-export const ONBOARD_CONTRACT_ADDRESS = "0x413370ed41FB9EE3aea0B1B91FD336cC0be1Bad6"
+export const ONBOARD_CONTRACT_ADDRESS = "0x60eA13A5f263f77f7a2832cfEeF1729B1688477c"
```
Comment (smiller-coti) [inline src/account/onboard-contract.ts]: Do we want to use deterministic deployment for testnet?

**Review by smiller-coti — CHANGES_REQUESTED** [review]

File: src/crypto_utils.ts (hunk 1 above)
Comment (gmesika-coti) [reply src/crypto_utils.ts]: lets go for it, as long as we are relasing today version

Comment (eitanz-coti) [reply src/crypto_utils.ts]: fixed.

**Comment by smiller-coti** [comment]
Just have to update the docs for the new function

**Pull request closed without merging**